api-server/src/roster/models.py

```diff
@@ -386,3 +386,17 @@ class UserImage(models.Model):
 
     def __str__(self):
         return f'{self.file.name or self.url}'
+
+class LeaveOfAbsence(models.Model):
```

Comment:
So im just curious how this is different than an event? Why not just use the same model and api just create another category?